src/librbd/api/PoolMetadata.h

```diff
@@ -4,6 +4,9 @@
 #ifndef CEPH_LIBRBD_API_POOL_METADATA_H
 #define CEPH_LIBRBD_API_POOL_METADATA_H
 
+#include <string>
```

Comment:
This is already included below -- and move `cstdint` down to the standard library includes below.